rebar.config

```diff
@@ -123,7 +123,15 @@
                                         "_build/prod/lib/aeutils/priv/epoch_config_schema.json",
                                         "data/epoch_config_schema.json"}]}
                            ]}
-                   ]}
+                   ]},
+            {system_test, [
+                {extra_src_dirs, ["system_test", "system_test/helpers"]},
+                {shell, [{apps, []}]},
+                {deps, [
+                    bbmustache,
+                    {hackney, ".*", {git, "git://github.com/benoitc/hackney.git", {branch, "master"}}}
```

Comment:
We should use a stable version of Hackney from Hex here, unless we have a policy to depend on Git repositories?

Reply:
At least versioned i.e. package version / repo tag / repo commit.